api/lib/getJSON.js

```diff
@@ -1,5 +1,45 @@
+const object = require('lodash/fp/object');
+
+const reduceObj = (accumulator, currentObj) => {
```

Comment:
some comments would be useful on why we need to to this. (we are going to forget some time down the line).